lib/pages/PickUpPage.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:carriage/widgets/AppBars.dart';
+import 'package:carriage/widgets/Buttons.dart';
+import 'package:carriage/widgets/Dialogs.dart';
+import 'package:carriage/widgets/RideInfoCard.dart';
+import 'package:flutter/material.dart';
+
+// TODO: replace with real model later
+class _TempPageData {
+  final String firstName;
+  final ImageProvider<dynamic> photo;
+  final DateTime time;
+  final String stop;
+  final String address;
+
+  _TempPageData(this.firstName, this.photo, this.time, this.stop, this.address);
+}
```

Comment:
Only define this once. Even though it is temporary, you already defined this in OnTheWayPage